ui/pages/liquidity.tsx

```diff
@@ -338,7 +338,12 @@ export default function Liquidity() {
                       className="btn btn-primary normal-case font-medium w-full"
                       action={withdrawAndClaimRewards}
                     >
-                      Withdraw all and claim
+                      <div
```

Comment:
The tooltip gets cut off
![Screen Shot 2022-08-13 at 9 16 48 PM](https://user-images.githubusercontent.com/17715009/184519726-6f0ccccd-13fa-4cd6-bda8-f060c19de972.png)

Reply:
Fixed tooltip overflow 👍